### ISSUE TITLE
maybe_update_lr in network_trainer.py()

### ISSUE THREAD

**Issue by xiaofang007** [MIC-DKFZ/nnUNet#1279]
Dear sir, I am looking at the code in network_trainer.py() because I find that even though my validation performance decrease, the learning rate does not update even though I use ReduceLROnPlateau. So may I ask in the comment the code says "lr scheduler is updated with moving average val loss" but in the implementation below, it is "self.lr_scheduler.step(self.train_loss_MA)" and it seems it is updated with moving average of train loss ? So may I ask about my confusion of this part of the code ? Thanks.

**Comment by Kobalt93** [comment]
Hi xiaofang007,
I think this has been already updated: https://github.com/MIC-DKFZ/nnUNet/blob/227d68e77f00ec8792405bc1c62a88ddca714697/nnunetv2/training/nnUNetTrainer/nnUNetTrainer.py#L883

**Comment by Kobalt93** [comment]
Hi @xiaofang007,
could you please try v2? There is a nice additional feature displaying not just the train/val curves, but also the epoch times and learning rates over the whole training.
